doc/tutorials/partition-mode.rst

```diff
@@ -0,0 +1,332 @@
+      |                    |                                                                |
+      |                    | size: host bar size rounded up to the nearest 4k               |
+      |                    |                                                                |
+      |                    | type: should always be set to PCIBAR_MEM32.                    |
+      |                    | For 64-bit memory space host BAR, it must be smaller than      |
+      |                    | 4GB in size, so that 32-bit memory UOS bar can always be used. |
+      |                    |                                                                |
+      |                    | I/O space bar is not supported so it can be left unspecified   |
+      +--------------------+----------------------------------------------------------------+
+      | pdev               | Specify the host bdf/bar for the attached PCI device.          |
+      |                    | For hostbridge, pdev is not used and can be left unspecified   |
+      +--------------------+----------------------------------------------------------------+
+
+
+
+Each PCI passthrough device is represented by pci_pdev structure:
+
+
+  .. table:: vpci_pdev
+      :widths: auto
+      :name: vpci_pdev
+
+      +--------------------+-------------------------------------------------------------+
+      | Member             | Description                                                 |
+      +====================+=============================================================+
+      | bdf                | Specify physical BDF for PCI passthrough device.            |
+      |                    | Use ``lspci`` command in SOS/native OS to select the PCI BDF|
+      |                    | you want to assign to UOS. Note that only PCI devices that  |
+      |                    | use MSI/MSI-X interrupts can be assigned to UOS, INTx is not|
+      |                    | supported                                                   |
+      +--------------------+-------------------------------------------------------------+
+      | bar                | Specify the host bar for the PCI passthrough device.        |
+      |                    | Use ``lspci -v`` command in SOS/native OS to locate the     |
+      |                    | entry for device and note down the host bar base and size,  |
+      |                    | and accordingly fill in the values in bar.                  |
+      |                    | I/O space bar is not supported so it can be left unspecified|
+      +--------------------+-------------------------------------------------------------+
+
+
+3. Do modifications for assigning processors to each UOS VM
+
+   A. In ``hypervisor/partition/vm_description.c``, change the following as they suit the platform resources.
+
+        ``NUM_USER_VMS`` - Denotes the number of UOS VMs booted by ACRN in partition mode.Change
```

Comment:
Indenting in reST like this doesn't generate correct HTML output.  Instead, write this as:

    A. In ``hypervisor/partition/vm_description.c``, change the following as they 
       suit the platform resources.

       ``NUM_USER_VMS``
          Denotes the number of UOS VMs booted by ACRN in partition mode. Change
          it according to the number of UOS VMs need to be booted in partition
          mode by ACRN.

       ``VM1_NUM_CPUS``
          Denotes the number of CPUs used by UOS VM1